rome/src/main/java/com/rometools/rome/io/impl/RSS20Parser.java

```diff
@@ -51,16 +50,10 @@ protected Description parseItemDescription(final Element rssRoot, final Element
 
     @Override
     public boolean isMyType(final Document document) {
-        return rootElementMatches(document) && versionMatches(document);
+        return rootElementMatches(document);
```

Comment:
This will only work as long as the RSS 2.0 parser is the last RSS parser that will be checked. As far as I know this is depending on the order in rome.properties. I would propose to check whether the version matches (like before) or whether the version is null.